server/perfkit/explorer/model/explorer_config_test.py

```diff
@@ -28,7 +28,10 @@ def testGetDefault(self):
         'default_dataset': explorer_config.DEFAULT_DATASET,
         'default_table': explorer_config.DEFAULT_TABLE,
         'analytics_key': explorer_config.DEFAULT_ANALYTICS_KEY,
-        'cache_duration': explorer_config.DEFAULT_CACHE_DURATION
+        'cache_duration': explorer_config.DEFAULT_CACHE_DURATION,
+        'restrict_save_to_admin': True,
+        'restrict_view_to_admin': True,
+        'restrict_query_to_admin': True
```

Comment:
Nit: please add trailing comma after the last entry, this is Python not JSON. Helps keep edits and diffs cleaner.